ansible/roles/base/tasks/main.yaml

```diff
@@ -1,25 +1,52 @@
 ---
+- set_fact:
+    sshd_allowusers: >-
+      {{ [ 'root' ] | union(sshd_allowusers_group | default([]))
+                    | union(sshd_allowusers_host  | default([])) }}
```

Comment:
sshd_allowgroups = `[] | union(sshd_allowgroups_host  | default([])) | union (sshd_allowgroups_group | default([]))`

Reply:
Automatically adding users to `sshd_allowgroup` becomes a bit tricky if we want to support multiple groups, though.